egret/model_library/unit_commitment/objective.py

```diff
@@ -71,6 +71,8 @@ def _add_shutdown_costs(model, add_shutdown_cost_var=True):
                                             'production_costs': None,
                                             'power_balance': None,
                                             'reserve_requirement': None,
+                                            'storage_service': None,
```

Comment:
To confirm, the changes to this file have nothing to do with fuel supply - correct? If so, I think it would be worth putting these into a distinct commit (for clarity / forensics).

Reply:
Right -- I think this would be difficult to excise from this PR at this point, but going forward I'll be sure to put clean-ups to the UC model in their own PR. These changes are in one commit, at least.